### PR TITLE
Feat: update meta tags

### PR DESCRIPTION
**Pull request by dyCodes**

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -1,7 +1,43 @@
...
-import type {AppProps} from "next/app";
-import Layout from "@src/components/atoms/Layout";
+import '@src/styles/globals.css';
+import type { AppProps } from 'next/app';
+import Layout from '@src/components/atoms/Layout';
+import Head from 'next/head';
 
-export default function App({Component, pageProps}: AppProps) {
-    return <Layout> <Component {...pageProps} /></Layout>
+export default function App({ Component, pageProps }: AppProps) {
```
Comment (Onyelaudochukwuka) [inline src/pages/_app.tsx:6]: Can use use next's metaData object here instead ?

**Comment by dyCodes** [comment]
@Onyelaudochukwuka

File: src/pages/_app.tsx (hunk 1 above)
Comment (Onyelaudochukwuka) [reply src/pages/_app.tsx:6]: @dyCodes

Comment (dyCodes) [reply src/pages/_app.tsx:6]: No, the metadata object is for the App router. We are using the pages router, that's why I used the Head component.

Comment (Onyelaudochukwuka) [reply src/pages/_app.tsx:6]: YEAH, I WAS THINKING WE COULD SWITCH SINCE WE JUST HAVE THE LANDING PAGE HERE

**Pull request merged**